test/performance/odp_bench_packet.c

```diff
@@ -126,6 +135,10 @@ typedef struct {
 	odp_time_t ts_tbl[TEST_REPEAT_COUNT];
 	/** Array for storing test data */
 	uint8_t data_tbl[TEST_REPEAT_COUNT][TEST_MAX_PKT_SIZE];
+	/** Options for exporting results */
+	test_common_options_t *common_options_ptr;
```

Comment:
This works, but probably it would be a bit safer to copy the whole struct here instead of using a pointer.